### PR TITLE
RHEL 8.10 builds and IDPF enablement for RHEL 8/8.10.

### PR DESCRIPTION
**Pull request by zmarano**

### CODE REVIEW

File: daisy_workflows/build-publish/enterprise_linux/almalinux_9.publish.json
Comment (a-crate) [inline daisy_workflows/build-publish/enterprise_linux/almalinux_9.publish.json:1]: Is this supposed to be alma 8?

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+cp /run/install/isodir/65-gce-disk-naming.rules /etc/udev/rules.d/
+cp /run/install/isodir/google_nvme_id /usr/lib/udev/
+chmod +x /usr/lib/udev/google_nvme_id
+# Wait for coldplug events from boot to settle, or we won't generate new events for the reload/trigger
+udevadm settle
+udevadm control --reload
+udevadm trigger --settle
+tee -a /tmp/disk-config << EOM
+# build_installer.py will replace with the id of the install disk to avoid race conditions
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg:33]: Don't disable predictable NIC names

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+cp /run/install/isodir/65-gce-disk-naming.rules /etc/udev/rules.d/
+cp /run/install/isodir/google_nvme_id /usr/lib/udev/
+chmod +x /usr/lib/udev/google_nvme_id
+# Wait for coldplug events from boot to settle, or we won't generate new events for the reload/trigger
+udevadm settle
+udevadm control --reload
+udevadm trigger --settle
+tee -a /tmp/disk-config << EOM
+# build_installer.py will replace with the id of the install disk to avoid race conditions
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg:33]: don't disable predictable nic names

File: daisy_workflows/build-publish/enterprise_linux/almalinux_9.publish.json
Comment (zmarano) [reply daisy_workflows/build-publish/enterprise_linux/almalinux_9.publish.json:1]: Actually no, its just IDPF for AlmaLinux 9 - we'll have to followup with Alma/Rocky 8 when that has been resolved.

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg (hunk 1 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg]: Done

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg (hunk 2 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg]: Done

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg [hunk 3]
```diff
@@ -30,7 +30,7 @@ udevadm control --reload
 udevadm trigger --settle
 tee -a /tmp/disk-config << EOM
 # build_installer.py will replace with the id of the install disk to avoid race conditions
-bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg:33]: extra `"`

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg [hunk 4]
```diff
@@ -30,7 +30,7 @@ udevadm control --reload
 udevadm trigger --settle
 tee -a /tmp/disk-config << EOM
 # build_installer.py will replace with the id of the install disk to avoid race conditions
-bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0"
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg:33]: extra `"`

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg (hunk 3 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg]: Done (pretty sure I did that last time too)

File: daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg (hunk 4 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap_byos.cfg]: Done

**Review by a-crate — APPROVED** [review]

**Pull request merged**